packages/tools/devtools/devtools-view/screenshot-tests.mjs

```diff
@@ -0,0 +1,147 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { decodeComponentId } from "@previewjs/api";
+import { createChromelessWorkspace } from "@previewjs/chromeless";
+import reactPlugin from "@previewjs/plugin-react";
+import chalk from "chalk";
+import { globby } from "globby";
+import * as path from "path";
+import { chromium } from "playwright";
+
+// The below implementation is derived from @previewjs/screenshot, but updated to allow parametric testing
+// of test "stories" in different theme contexts.
+// https://github.com/fwouts/previewjs/blob/main/screenshot/src/index.ts
+
+/**
+ * Pattern used to discover test "story" modules for screenshot tests.
+ */
+const storyFilePathPattern = "src/test/screenshot-tests/*.stories.tsx";
+
+/**
+ * Output directory, relative to the test "story", under which the generated screenshots will be saved.
+ */
+const outputDirectoryPath = "src/test/screenshot-tests/__screenshots__";
+
+/**
+ * The supported themes in which each test "story" will be rendered.
+ * A separate screenshot will be generated for each of these.
+ */
+const themes = ["dark", "light", "high-contrast"];
```

Comment:
Can we also add a note about how these don't need to match the constants in ThemeHelper?